### ISSUE TITLE
Some extensions do not work

### ISSUE THREAD

**Issue by RobDangerous** [Kode/KodeStudio#48]
In particular https://marketplace.visualstudio.com/items?itemName=heyimfuzz.banner-comments and https://marketplace.visualstudio.com/items?itemName=alefragnani.Bookmarks

**Comment by glenngartner** [comment]
I'll add another one: VIM extension, found here: https://marketplace.visualstudio.com/items?itemName=vscodevim.vim

After I downloaded the extension from the VSCode marketplace as a .vsix file, I ran the "F1>Extensions: Install from VSIX" command within Kode Studio, and Kode returns a successful install prompt. The extension was unpacked and installed in my "C:\Users\me\.kodestudio\extensions" folder in Windows.  Still, it doesn't activate, even after a restart of Kode.

This extension would be a pleasant productivity enhancer for me.
